blogfrontend/src/components/Login.js

```diff
@@ -0,0 +1,59 @@
+import React,{useState} from 'react'
+import { Link } from "react-router-dom";
+
+const Login = () => {
+    const [username,setUserName]=useState({
```

Comment:
the naming convention is not proper as you are assigning object and the variable name is username it should be userDetails.